keps/sig-cluster-lifecycle/0035-20190128-addons-via-operators.md

```diff
@@ -0,0 +1,181 @@
+KEP: Addons via Operators
+
+---
+kep-number: 35
+title: Addons via Operators
+authors:
+  - "@justinsb"
+owning-sig: sig-cluster-lifecycle
+reviewers:
+  - TBD
+approvers:
+  - TBD
```

Comment:
Assign me as both reviewer and approver. 
also 
/assign @luxas @roberthbailey @timothysc